app/templates/views/accessibility_statement.html

```diff
@@ -0,0 +1,179 @@
+  </p>
+
+  <h2 class="heading-medium">Technical information about this website’s accessibility</h2>
+
+  <p class="govuk-body">
+    GDS is committed to making its website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+  </p>
+
+  <p class="govuk-body">
+    GOV.UK Notify is partially compliant with the Web Content Accessibility Guidelines version 2.1 AA standard, due to the non-compliances listed below.
+  </p>
+
+  <h2 class="heading-medium">Non-accessible content</h2>
+
+  <p class="govuk-body">
+    The content listed below is non-accessible for the following reasons.
+  </p>
+
+  <h3 class="heading-small">Non compliance with the accessibility regulations</h3>
+
+  <p class="govuk-body">
+    Low vision users cannot access the table we use to show CSV data when viewed at high zoom. This fails <a class="govuk-link" href="https://www.w3.org/TR/WCAG21/#reflow">WCAG 1.4.10 success criteria (Reflow)</a>. We plan to fix this by the end of 2020.
+  </p>
+
+  <p class="govuk-body">
+    Screen reader users may find the process of moving templates and folders confusing. This fails <a class="govuk-link" href="https://www.w3.org/TR/WCAG21/#name-role-value">WCAG 4.1.2 success criteria (Name, role, value)</a>. We plan to fix this by November 2020.
+  </p>
+
+  <p class="govuk-body">
+    Screen reader users may find it confusing that a specific part of some pages is focused when they load. This fails <a class="govuk-link" href="https://www.w3.org/TR/WCAG21/#focus-order">WCAG 2.4.3 success criteria (Focus Order)</a>. We plan to fix this by October 2020.
+  </p>
+
+  <h3 class="heading-small">
+    PDFs and other documents
+  </h3>
+
+  <p class="govuk-body">
+    One page links to a PDF documents that is not fully accessible. This is not a fail under any WCAG 2.1 success criterion.</p>
+
+  <p class="govuk-body">
+    We provide a written description of the information in the PDF on an HTML page as an accessible alternative.
+  </p>
+
+  <h3 class="heading-small">
+    Emails
+  </h3>
+
+  <p class="govuk-body">
+    The NVDA screen reader announces the URL of the GOV.UK logo image in emails when viewed in Microsoft Outlook. This has been raised as <a class="govuk-link" href="https://github.com/alphagov/reported-bugs/issues/46">an issue on NVDA’s issue tracker</a>.
+  </p>
+
+  <p class="govuk-body">
+    The JAWS and NVDA screen readers announce extra spaces in emails when viewed in Microsoft Outlook.
+  </p>
+
+  <p class="govuk-body">
+    We’ll continue to investigate the cause of these issues and, if possible, fix them by March 2021.
+  </p>
+
+  <h3 class="heading-small">
+    Status page
+  </h3>
+
+  <p class="govuk-body">
+    The following content on the Notify status page is not compliant with the Web Content Accessibility Guidelines version 2.1 AA standard:
```

Comment:
I’d suggest adding a link here so people can confirm which page we’re talking about, like we do when talking about the documentation domain at the top of the page:
```suggestion
    The following content on the <a class="govuk-link" href="https://status.notifications.service.gov.uk/">Notify status page</a> is not compliant with the Web Content Accessibility Guidelines version 2.1 AA standard:
```